payroll/charts.py

```diff
@@ -6,6 +6,9 @@
 
 
 class ChartHelperMixin(object):
+    def _get_bar_color(self, lower_edge, upper_edge):
+        return '#004c76'
```

Comment:
I wonder if there's a better place to stick these hex values. Maybe in the settings file or something? Just seems like maybe the wrong place to stick styling info.